openff/evaluator/protocols/reweighting.py

```diff
@@ -491,7 +491,7 @@ def _reweight_observables(
             observable_dimensions = observable.value.shape[1]
             assert observable_dimensions == 1
 
-            results = mbar.computeExpectations(
+            results = mbar.compute_expectations(
                 observable.value.T.magnitude,
```

Comment:
We need to support both versions for compatibility with some external packages. I have a trick for handling this that I can take care of elsewhere. I thought I did, but this must not be run in the normal test suite - any other occurrences you noticed?

Reply:
I'm working on this in https://github.com/openforcefield/openff-evaluator/pull/479